pyproject.toml

```diff
@@ -52,10 +51,10 @@ django-split-settings = "^1.2.0"
 pexpect = "^4.9.0"
 ansible-runner = ">=2.3"
 python-gnupg = "^0.5.2"
-autobahn = {git = "https://github.com/crossbario/autobahn-python.git", rev = "v24.4.2"}
+autobahn = { git = "https://github.com/crossbario/autobahn-python.git", rev = "v24.4.2" }
 psycopg = "^3.1.17"
 xxhash = "*"
-pyjwt = { version="*", extras = ["cryptography"] }
+pyjwt = { version = "*", extras = ["cryptography"] }
```

Comment:
@Alex-Izquierdo This should be crypto instead of cryptography then we can close PR #985

Reply:
updated